protocol/contracts/lpp/src/stub/mod.rs

```diff
@@ -26,26 +25,37 @@ pub mod loan;
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[cfg_attr(any(test, feature = "testing"), derive(Eq, PartialEq))]
-pub struct LppRef {
+pub struct LppRef<Lpns> {
     addr: Addr,
-    currency: SymbolOwned,
+    lpn: SymbolOwned,
```

Comment:
good intention ... but have to come up with a migration plan of the existing data